### DISCUSSION TITLE
Images and rotation

### DISCUSSION THREAD

**Discussion by abdmh** [slint-ui/slint#863] [General] [answered]
Can images be rotated/animated in v 0.1.6?

**Comment by hunger** [comment, marked as answer]
You can animate the `colorize` property, which applies tinting to the image. You can also animate all the usual properties like position and width/height.

Rotation is unfortunately not possible yet: Rotation is a pretty heavy operation without hardware backing it. So we are thinking on how to support this in a useful way that does not break too much of the embedded use case. Maybe we could have custom "Rotation Elements" that can be used to rotate things? That would at least make it very obvious where rotation is used -- and it can easily be avoided in designs targeting low-powered devices.

**Comment by ogoffart** [comment]
See also previous discussion: https://github.com/sixtyfpsui/sixtyfps/discussions/709

**Reply by abdmh** [reply]
Noted with thanks
